examples/babi_memnn.py

```diff
@@ -27,6 +28,11 @@
 import re
 
 
+if K.backend() == 'mxnet':
```

Comment:
Without adding this check, our backend is already showing a clear error message:
```
NotImplementedError: MXNet Backend: Sparse operations are not supported yet.
```
So I think we can avoid repeated check, and its on our list of not supported example [issue]
(https://github.com/awslabs/keras-apache-mxnet/issues/79) 

I was adding this check for other examples only if error messaging is not clear or does not indicate root cause.

Reply:
makes sense removed.